bench/wtperf/wtperf_run_py/wtperf_run.py

```diff
@@ -197,6 +192,14 @@ def setup_perf_stats():
                                  pattern=r'Executed \d+ update operations',
                                  input_offset=1,
                                  output_label='Update count'))
+    perf_stats.add_stat(PerfStatMax(short_label="max_update_throughput",
```

Comment:
The introduction of PerfStatMin and PerfStatMax is a good change! 
For [PEP8 compliance](https://www.flake8rules.com/rules/E128.html), the parameters in lines 196-198 and 200-202 should be indented to align with 'short_label)